mdtraj/formats/pdb/pdbfile.py

```diff
@@ -507,10 +502,17 @@ def _write_footer(self):
             "DI",
             "HOH",
         ]
+
+    def _write_footer(self):
+        if not self._mode == "w":
+            raise ValueError("file not opened for writing")
+
+        # Identify bonds that should be listed as CONECT records.
+
         conectBonds = []
         if self._last_topology is not None:
             for atom1, atom2 in self._last_topology.bonds:
-                if atom1.residue.name not in standardResidues or atom2.residue.name not in standardResidues:
+                if atom1.residue.name not in self._standardResidues or atom2.residue.name not in self._standardResidues:
```

Comment:
Here, `standardResidues` has been moved from being defined in the footer to being a class attribute on `PDBFile`. This is done so that it is more accessible for PDBx for header and footer reading and writing.

Considering the need to share this across both file types, I'd recommend moving it out of PDBFile entirely and importing the list to both objects. This way, `PDBFile` is not imported to `PDBxFile`. I would recommend putting this in a module in the formats directory and importing it to both PDB and PDBx

Reply:
I see now that there is something similar occurring for at least one static method as well. I am going to think about this a little more